tests/50federation/30room-join.pl

```diff
@@ -12,13 +9,18 @@ sub make_auth_events
       my ( $user, $inbound_server ) = @_;
       my $local_server_name = $inbound_server->server_name;
 
+      my $creator = '@50fed:' . $local_server_name;
+
+      my $room = SyTest::Federation::Room->create(
+         server  => $inbound_server,
```

Comment:
As per discussion on matrix, this is quite a strange line of code which I don't know how to interpret.